src/components/Login/LoginFeatures.js

```diff
@@ -33,46 +34,51 @@ const LoginFeatures = (props) => {
           setPassword(e.target.value);
         }}
       />
-      <Button value="Login" size="large" />
-      {/* {props.error && <Error errMessage={props.error} />} */}
-      <div>
-        <Button
-          variant="contained"
-          fullWidth={true}
-          gradient="true"
-          size="large"
-          onClick={logInfo}
-        >
-          LOGIN
-        </Button>
-      </div>
+
+      {"errorShow" && (
+        <Error errorMsg={"errorShow"} className="error-container" />
+      )}
```

Comment:
You have it as `"errorShow"` instead of `errorShow` (in 2 places), so it's always displaying `errorShow` as an error message.